import_ldraw.py

```diff
@@ -186,9 +173,14 @@ def __init__(self, context, filename, mat, colour=None):
                     f.material_index = me.materials.find(material.name)
 
             self.ob = bpy.data.objects.new('LDrawObj', me)
-            self.ob.name = os.path.basename(filename)
+            # Naming of objects: filename of .dat-file, without extension
+            self.ob.name = os.path.basename(filename.split('.')[0])
```

Comment:
I was thinking about this line last night, and I noted that file names always end in .dat or .ldr. So a cleaner way to write this would be to use string slicing.

```python
self.ob.name = os.path.basename(filename)[:-4]
```